src/Catty/ViewController/Continue&New/MaintainObject/MaintainSounds/SoundsTableViewController+SelectFile.swift

```diff
@@ -0,0 +1,79 @@
+/**
+ *  Copyright (C) 2010-2021 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+import MobileCoreServices
+
+extension SoundsTableViewController {
+
+    @objc
+    func showSoundsSelectFile() {
+        let documentPicker = UIDocumentPickerViewController(
+            documentTypes: ["public.wav", "public.mpga", "public.mp3", "public.ogg"],
+            in: .import)
+        if #available(iOS 11.0, *) {
+            documentPicker.allowsMultipleSelection = true
+        }
+
+        documentPicker.delegate = self
+        documentPicker.modalPresentationStyle = .formSheet
+
+        present(documentPicker, animated: true)
+    }
+
+    private func showImportAlert(itemName: String) {
```

Comment:
Since you used the `self.showDownloadSoundAlert(sound)` function from the `SoundsTableViewController+MediaLibrary.swift` below (line 68), why do you need the redefinition of the `showImportAlert(itemName: String)` function? Can't you use the showImportAlert function of the `SoundsTableViewController+MediaLibrary.swift`?